### PR TITLE
[HID] Enable Keyboard Passthru

### PR DESCRIPTION
**Pull request by Gliniak**

### CODE REVIEW

File: src/xenia/hid/winkey/winkey_input_driver.cc [hunk 1]
```diff
@@ -25,10 +25,29 @@
...
+             "slot to be interpreted as controller. Possible values: -1 - "
+             "Disabled (Keyboard is in "
+             "gamepad mode), [0, 3] - Keyboard is assigned as VK for that user",
+             "HID");
+
 namespace xe {
 namespace hid {
 namespace winkey {
 
+bool __inline IsPassThruForUserEnabled(uint32_t user_index) {
```
Comment (ghost) [inline src/xenia/hid/winkey/winkey_input_driver.cc]: can't this just be user_index == cvars::keyboard_passthru_user_index
is -1 possible? it doesnt seem like it should be

Comment (Gliniak) [reply src/xenia/hid/winkey/winkey_input_driver.cc]: Yeah it shouldn't. -1 is used but as 0xFF to define any user

**Comment by nickcool2901** [comment]
keyboard_passthru isnt in any new builds

**Comment by Rackover** [comment]
Would love to see merged in, is there any problem with the PR that is currently delaying it ?

**Comment by Gliniak** [comment]
> Would love to see merged in, is there any problem with the PR that is currently delaying it ?

Yes, It's not fully functional and doesn't work in some cases

**Comment by Rackover** [comment]
I guess nothing ever is, but it's a start! If it's not breaking any other functionality in the emulator, maybe this could get merged in and improved later? 
Of course if it's breaking existing functionality it should not, but maybe it's worth having if it's not.

**Comment by ItsJokerZz** [comment]
How do you enable? Downloaded the PR build `f6ae651_canary_experimental` and see no option...

**Pull request merged**